File: Media-Pembelajaran-Administrasi-Infrastruktur-Jaringan-master/app/src/main/java/com/fano/administrasiinfrastrukturjaringan/SilabusActivity.java
package com.fano.administrasiinfrastrukturjaringan;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;
import android.widget.TextView;

import com.fano.administrasiinfrastrukturjaringan.model.DataKd;
import com.github.barteksc.pdfviewer.PDFView;

public class SilabusActivity extends AppCompatActivity {
    PDFView pdfView;
    ImageButton ibBack;
    TextView tvToolbar;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_silabus);

        pdfView = (PDFView) findViewById(R.id.pdfView);
        tvToolbar = findViewById(R.id.tv_toolbar);
        ibBack = findViewById(R.id.ib_back);

        ibBack.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                onBackPressed();
            }
        });

        pdfView.fromAsset("silabus.pdf").load();
        tvToolbar.setText("Silabus");

    }
}

File: AdministrasiSistemJaringan/app/src/main/java/id/topan/administrasisistemjaringan/SilabusActivity.kt
package id.topan.administrasisistemjaringan

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.webkit.WebView

class SilabusActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_silabus)

        val web: WebView = findViewById(R.id.webSilabus)

        web.requestFocus()
        web.settings.javaScriptEnabled = true
        val myPdfUrl = "pdf.pdf"
        val url = "https://docs.google.com/viewer?embedded=true&url=jualtbs.com/$myPdfUrl"
        web.loadUrl(url)
    }
}